indico/modules/events/papers/templates/reviewing/timeline.html

```diff
@@ -15,6 +15,23 @@
     </ul>
 {% endmacro %}
 
+{% macro _render_paper_files(files, spotlight_file=None) %}
+    <ul class="paper-files flexrow f-wrap">
+        {% for file in files %}
+            {% if file != spotlight_file %}
+                {% set icon = icon_from_mimetype(file.content_type, 'icon-file-filled') %}
+                <li>
+                    <a class="attachment paper-file i-button text-color outline dashed {{ icon }}"
+                       href="{{ url_for('.download_file', file) }}"
```

Comment:
If you happen to follow [@ThiefMaster's advice](https://github.com/indico/indico/pull/2859/files#r103702880), it applies here as well.